arbitrum-docs/for-devs/troubleshooting-building.md

```diff
@@ -1,11 +1,11 @@
 ---
 title: 'Troubleshooting: Building Arbitrum dApps'
+sidebar_label: 'Troubleshooting: Building Arbitrum dApps'
+description: List of questions and answers frequently asked by developers
+reader_audience: developers who want to build on Ethereum/Arbitrum
+content-type: faq
 ---
 
-import FaqPartial, { toc as FAQTOC } from '../partials/_troubleshooting-building-partial.md';
+import FaqPartial from '../partials/_troubleshooting-building-partial.md';
 
-<div data-faq-origin-slug="build-faq">
```

Comment:
@symbolpunk , you might notice I removed this structure as I didn't see it being used anywhere. Can you confirm that we can simplify that structure as I did?
Thanks! 🙏

Reply:
LGTM!

FYI, the original design assumed that we'd

 1. Generate a mono-FAQ that contains all FAQs on Notion
 2. Embed the entire mono-FAQ on sub-pages
 3. Use `data-faq-origin-slug` to hide irrelevant FAQs via CSS
 
 ...the idea being that we can include/exclude FAQs from pages by adding/removing slug tags on Notion instead of dealing with partial markdown files. But I don't think we leaned into this approach, and I don't think we need to :).

Reply:
Understood. Yeah, we are already filtering out the FAQs in Notion, and it might make sense the have the filter there (in Notion).
If we ever change the system, we can add the wrappers back.
Thanks!